frontend/public/components/nav/admin-nav.tsx

```diff
@@ -349,10 +349,10 @@ const AdminNav = () => {
         <ResourceNSLink
```

Comment:
name={t('nav~Cluster Settings')}, if this nav needs to be CamelCased?

Reply:
I left it un-CamelCased because we didn't have a model for it. Sam is out, but @rhamilto said this is ok to leave as-is. It's not a Kubernetes term.